### ISSUE TITLE
Bug in VRTK_InteractTouch.cs makes Unity stop responding

### ISSUE THREAD

**Issue by Backer-Sultan** [thestonefox/VRTK#1231]
Source of VRTK: Github
  Version of VRTK: b5759f176e8c87adf237121063ab9fc6573ac22a
  Version of Unity3D: 5.5.2f
  Hardware used Oculus Rift
  SDK used OpenVR

Steps to reproduce
1. Create a 3D object with a collider.
2. Add VRTK_InteractTouch.cs to it.
3. Assign the object itself to VRTK_InteractTouch.cs's field 'CustomColliderContainer'.
4. Enter Play mode.

Expected behavior
VRTK_InteractTouch.cs accepts the assigned GameObject.

Current behavior
Unity shows the following message:
"Unity Editor has stopped working.
A problem caused the program to stop working correctly. Windows will close the program and notify you if a solution is available." 
with 2 options: Debug, Close Program.

**Comment by thestonefox** [comment]
You're using it wrong. Interact touch goes on the controller script alias.

**Comment by Backer-Sultan** [comment]
@thestonefox I'm using custom hands and trying to figure out how to override the default behavior without modifying any script. I tried delete the controller alias and add 'VRTK_InteractToutch.cs' and 'VRTK_ControllerEvents.cs' to the hands I'm using instead. I'm sting getting the default colliders so I assigned the colliders of the hands I'm using to 'CustomColliderContainer' and got the error.
Any suggestion of how to solve this?
Thanks!
